qcengine/programs/adcc.py

```diff
@@ -0,0 +1,133 @@
+            If raise_error is True and adcc or psi4 are missing, the error message is raised.
+        """
+        found_adcc = which_import(
+            "adcc",
+            return_bool=True,
+            raise_error=raise_error,
+            raise_msg="Please install via `conda install adcc -c adcc`.",
+        )
+        found_psi4 = which_import(
+            "psi4",
+            return_bool=True,
+            raise_error=raise_error,
+            raise_msg="Please install psi4 for adcc harness via `conda install psi4 -c psi4`.",
+        )
+        return found_adcc and found_psi4
+
+    def get_version(self) -> str:
+        """Return the currently used version of adcc"""
+        self.found(raise_error=True)
+
+        which_prog = which_import("adcc")
+        if which_prog not in self.version_cache:
+            import adcc
+
+            self.version_cache[which_prog] = safe_version(adcc.__version__)
+        return self.version_cache[which_prog]
+
+    def compute(self, input_model: "AtomicInput", config: "TaskConfig") -> "AtomicResult":
+        """
+        Runs adcc
+        """
+        self.found(raise_error=True)
+
+        import adcc
+        import psi4
+
+        mol = input_model.molecule
+        model = input_model.model
+        conv_tol = input_model.keywords.get("conv_tol", 1e-6)
+
+        psi4_molecule = psi4.core.Molecule.from_schema(dict(mol.dict(), fix_symmetry="c1"))
+        psi4.core.clean()
+        psi4.core.be_quiet()
+        psi4.set_options(
+            {
+                "basis": model.basis,
+                "scf_type": "pk",
+                "e_convergence": conv_tol / 100,
+                "d_convergence": conv_tol / 10,
+                # 'maxiter': max_iter,
+                "reference": "RHF" if mol.molecular_multiplicity == 1 else "UHF",
+            }
+        )
+        _, wfn = psi4.energy("HF", return_wfn=True, molecule=psi4_molecule)
+        adcc.set_n_threads(config.ncores)
+        compute_success = False
+        try:
+            adcc_state = adcc.run_adc(wfn, method=model.method, **input_model.keywords)
+            compute_success = adcc_state.converged
+        except adcc.InputError as e:
+            raise InputError(str(e))
+        except Exception as e:
+            raise UnknownError(str(e))
+
+        input_data = input_model.dict(encoding="json")
+        output_data = input_data.copy()
+        output_data["success"] = compute_success
+
+        if compute_success:
+            mp = adcc_state.ground_state
+            # MP(3) energy for CVS-ADC(3) calculations is still a missing feature in adcc
+            # ... we store this variant here to be able to fall back to MP(2) energies.
+            is_cvs_adc3 = adcc_state.method.level >= 3 and mp.has_core_occupied_space
+            mp_energy = mp.energy(adcc_state.method.level if not is_cvs_adc3 else 2)
+            output_data["return_result"] = mp_energy
```

Comment:
```suggestion
            output_data["return_result"] = adcc_state.excitation_energy[0]
```

I think this would be better, if it's ok with you. There isn't much excited state precedent to follow, and Psi4 should probably change to match, but it's fairly rigorous in QCA that `return_result` corresponds with `model.method`, so in the absence of a target state, lowest excited best. If there was a CIS, I'd push lowest excited, too, rather than HF GS energy.

Reply:
Done.